src/common/regions/preprocessed_regions_data.ts

```diff
@@ -0,0 +1,33 @@
+import keyBy from 'lodash/keyBy';
+import values from 'lodash/values';
+import mapValues from 'lodash/mapValues';
+
+import statesByFipsJson from 'common/data/states_by_fips.json';
+import countiesByFipsJson from 'common/data/counties_by_fips.json';
+import metroAreasByFipsJson from 'common/data/metro_areas_by_fips.json';
+
+import { State, County, MetroArea } from './types';
+
+export const statesByFips = mapValues(statesByFipsJson, (v: any) => {
```

Comment:
It'd be better to just drop the `: any` type and do `v => {` so TypeScript infers the type rather than specifying it as `any` which means TypeScript won't do any type-checking on it.

Reply:
it's a little tricky with the types inferred from the loaded JSON files, but I'll figure something out.

Reply:
I'm getting a lot of: 
```
$ tsc
src/common/regions/preprocessed_regions_data.ts(18,14): error TS2322: Type 'Dictionary<boolean>' is not assignable to type '{ [fips: string]: State; }'.
  Index signatures are incompatible.
    Type 'boolean' is not assignable to type 'State'.
src/common/regions/preprocessed_regions_data.ts(25,14): error TS2322: Type 'Dictionary<boolean>' is not assignable to type '{ [fips: string]: County; }'.
  Index signatures are incompatible.
    Type 'boolean' is not assignable to type 'County'.
src/common/regions/preprocessed_regions_data.ts(32,14): error TS2322: Type 'Dictionary<boolean>' is not assignable to type '{ [fips: string]: MetroArea; }'.
  Index signatures are incompatible.
    Type 'boolean' is not assignable to type 'MetroArea'.
error Command failed with exit code 1.

```

with an attempt that currently looks like this, if you can help me sort it out : (
```export const statesByFips: {[fips: string]: State} = mapValues(
  statesByFipsJson, State.fromObject);
  /*(v: StateObject): State => {
    return State.fromObject(v);
  },
);
*/
export const countiesByFips: {[fips: string]: County} = mapValues(
  countiesByFipsJson,
  (v: CountyObject): County => {
    return County.fromObject(v, statesByFips);
  },
);

export const metroAreasByFips: {[fips: string]: MetroArea} = mapValues(
  metroAreasByFipsJson,
  (v: MetroAreaObject): MetroArea => {
    return MetroArea.fromObject(v, statesByFips, countiesByFips);
  },
);
```

Reply:
I think this is probably why I went with 'any' the first time around :lolsob:

Reply:
oh I got it figured out, I needed to cast the imported JSON in to the expected type explicitly, then typescript could do its thing.